files/ko/web/javascript/reference/global_objects/promise/allsettled/index.html

```diff
@@ -43,11 +92,14 @@ <h2 id="문서_표준">문서 표준</h2>
  </tbody>
 </table>
 
-<h2 id="지원_브라우저">지원 브라우저<p>{{Compat("javascript.builtins.Promise.allSettled")}}</p>
+<h2 id="browser_compatibility">지원 브라우저
```

Comment:
[browser_compatibility 단어 명세 참고](https://github.com/mdn/translated-content/issues/1631) 및 태그가 닫히지 않았습니다.

```suggestion
<h2 id="browser_compatibility">브라우저 호환성</h2>
```